package.json

```diff
@@ -19,6 +19,7 @@
     "css-loader": "^6.11.0",
     "elasticlunr": "^0.9.5",
     "gatsby": "^5.13.3",
+    "gatsby-link": "^5.13.1",
```

Comment:
Is there a reason it was added?
Please add reason for change in `Additional Considerations`

Reply:
I added a comment